src/web/mjs/connectors/ScantradUnion.mjs

```diff
@@ -27,11 +39,11 @@ export default class ScantradUnion extends Connector {
         return data.map(element => {
             let number = element.querySelector('span.chapter-number').innerText.trim();
             let title = element.querySelector('span.chapter-name').innerText.trim();
+            title.length ? title = ' - ' + title : title;
```

Comment:
May use ternary operator for assignment
`title = title.length ? ' - ' + title : title`